src/main/kotlin/no/nav/familie/ef/sak/vilkår/VurderingController.kt

```diff
@@ -102,4 +104,29 @@ class VurderingController(
         gjenbrukVilkårService.gjenbrukInngangsvilkårVurderinger(request.behandlingId, request.kopierBehandlingId)
         return Ressurs.success(vurderingService.hentEllerOpprettVurderinger(request.behandlingId))
     }
+
+    @PostMapping("gjenbruk-enkelt-vilkar")
+    fun gjenbrukEnkeltVilkår(
+        @RequestBody request: EnkeltVilkårForGjenbrukRequest,
+    ): Ressurs<VilkårsvurderingDto> {
+        val behandlingForGjenbruk = gjenbrukVilkårService.finnBehandlingerForGjenbruk(request.behandlingId).first()
+
+        tilgangService.validerTilgangTilBehandling(behandlingForGjenbruk.id, AuditLoggerEvent.ACCESS)
+        tilgangService.validerTilgangTilBehandling(request.behandlingId, AuditLoggerEvent.UPDATE)
+        tilgangService.validerHarSaksbehandlerrolle()
+
+        val vilkårsVurderingForGjenbruk = gjenbrukVilkårService.gjenbrukInngangsvilkårVurdering(request.behandlingId, behandlingForGjenbruk.id, request.vilkårId)
+        return Ressurs.success(vilkårsVurderingForGjenbruk.tilDto())
+    }
+
+    @GetMapping("{behandlingId}/gjenbrukbare-vilkar")
+    fun hentAlleGjenbrukbareVilkårsvurderinger(
+        @PathVariable behandlingId: UUID,
+    ): Ressurs<List<UUID>> {
+        val behandlingForGjenbruk = gjenbrukVilkårService.finnBehandlingerForGjenbruk(behandlingId).first()
+        tilgangService.validerTilgangTilBehandling(behandlingForGjenbruk.id, AuditLoggerEvent.ACCESS)
+        tilgangService.validerTilgangTilBehandling(behandlingId, AuditLoggerEvent.UPDATE)
```

Comment:
Dette er vel ACCESS, ikke UPDATE

Reply:
På linje 101, i post-endepunktet "gjenbruk" så behøver man vel heller ikke å sjekke tilgang både for access og update. Jeg tror linje 101 kan fjærnes

Reply:
Jeg tenker dette blir riktig - (ish)

Reply:
Ref. diskusjonen over så blir det riktig at man sjekker både access og update allikevel 👍